cas-server-webapp/src/main/webapp/WEB-INF/spring-configuration/authenticationDelegation.xml

```diff
@@ -56,6 +63,7 @@
                 <ref bean="orcid" />
                 <ref bean="okstate" />
                 <ref bean="callutheran" />
+                <ref bean="cord" />
```

Comment:
* Step 2: Don't forget to add the newly created bean to the client lists. Otherwise, it is disabled. I made the mistake this time ...